packages/appstore/src/components/platform-launcher/platform-launcher.tsx

```diff
@@ -1,35 +1,81 @@
             <div className='platform-launcher__container'>
                 <div className='platform-launcher__container--icon'>
-                    <WalletIcon icon={app_icon} />
+                    <WalletIcon icon={icon} />
                 </div>
-                <div className='platform-launcher__container--title-desc-wrapper'>
-                    <Text className='platform-launcher__container--title-desc-wrapper--title' weight='bold'>
-                        <Localize i18n_default_text={app_title} />
-                    </Text>
-                    <Text className='platform-launcher__container--title-desc-wrapper--description'>
-                        <Localize i18n_default_text={app_desc} />
-                    </Text>
-                </div>
-            </div>
-            <div className='platform-launcher__trade-button'>
-                <Button primary small onClick={undefined} type='button'>
-                    <Localize i18n_default_text='Trade' />
-                </Button>
+                <DesktopWrapper>
+                    <div className='platform-launcher__container--title-desc-wrapper'>
+                        <Text
+                            size='xxs'
+                            line_height='l'
+                            className='platform-launcher__container--title-desc-wrapper--title'
+                            weight='bold'
+                        >
+                            <Localize i18n_default_text={title} />
+                        </Text>
+                        <Text
+                            size='xxs'
+                            line_height='l'
+                            className='platform-launcher__container--title-desc-wrapper--description'
+                        >
+                            <Localize i18n_default_text={description} />
+                        </Text>
+                    </div>
+                </DesktopWrapper>
+                <MobileWrapper>
+                    <div className='platform-launcher__container--title-desc-wrapper'>
+                        <Text
+                            size='xxs'
+                            line_height='s'
+                            className='platform-launcher__container--title-desc-wrapper--title'
+                            weight='bold'
+                        >
+                            <Localize i18n_default_text={title} />
+                        </Text>
+                        <Text
+                            size='xxxs'
+                            line_height='l'
+                            className='platform-launcher__container--title-desc-wrapper--description'
+                        >
+                            <Localize i18n_default_text={description} />
+                        </Text>
+                    </div>
+                </MobileWrapper>
             </div>
+            {has_real_account && (
+                <>
```

Comment:
```suggestion
                <React.fragment>
```

Reply:
@mahdiyeh-fs  It is just short form of   <React.fragment> right?

Reply:
yes, but we are using the <React.fragment> approach in deriv-app